apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -1045,6 +1058,11 @@ namespace MWPhysics
         return physactor && physactor->getOnGround();
     }
 
+    bool PhysicsSystem::isIdle(const MWWorld::Ptr &actor)
```

Comment:
'isIdle' seems like a misnomer. What you're really using it for is on ground status. And if you'd called it onGround, you might have noticed that we already have a PhysicActor::mOnGround. Why not use that?

Reply:
> we already have a PhysicActor::mOnGround. Why not use that?

IIRC, to update mOnGround, we should run physics calculations for actor, but the point is not to run physics calculations for actor.

About misnomer: I agree, but can not find a good name. Any suggestions?